### PR TITLE
Fix globals, part 16: `clearInterval()` & `setInterval()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the https://github.com/mdn/content/issues/33019

/cc @wbamberg

### CODE REVIEW

File: files/en-us/web/api/workerglobalscope/setinterval/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+See the same name section under {{domxref("Window.setInterval", "setInterval()")}} for examples.
+
+## Description
+
+See the same name section under {{domxref("Window.setInterval", "setInterval()")}} for description.
+
+## Usage notes
+
+See the same name section under {{domxref("Window.setInterval", "setInterval()")}} for usage notes.
```
Comment (wbamberg) [inline files/en-us/web/api/workerglobalscope/setinterval/index.md:62]: I would omit these, and just have a sentence at the end of the page intro, just before `## Syntax`, something like:

> This method is also available in window contexts: for a detailed description of `setInterval()` usage, see the  {{domxref("Window.setInterval()")}} page.

File: files/en-us/web/api/workerglobalscope/setinterval/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+It may be helpful to be aware that `setInterval()` and {{domxref("setTimeout()")}} share the same pool of IDs, and that `clearInterval()` and {{domxref("clearTimeout", "clearTimeout()")}} can technically be used interchangeably.
+For clarity, however, you should try to always match them to avoid confusion when maintaining your code.
+
+> [!NOTE]
+> The `delay` argument is converted to a signed 32-bit integer.
+> This effectively limits `delay` to 2147483647 ms, roughly 24.8 days, since it's specified as a signed integer in the IDL.
+
+## Examples
+
+See the same name section under {{domxref("Window.setInterval", "setInterval()")}} for examples.
```
Comment (wbamberg) [inline files/en-us/web/api/workerglobalscope/setinterval/index.md:54]:
```suggestion
See {{domxref("Window.setInterval", "setInterval()")}} for examples.
```

File: files/en-us/web/api/workerglobalscope/setinterval/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+slug: Web/API/WorkerGlobalScope/setInterval
+page-type: web-api-instance-method
+browser-compat: api.setInterval
+---
+
+{{APIRef("HTML DOM")}}{{AvailableInWorkers("worker")}}
+
+The **`setInterval()`** method of the {{domxref("WorkerGlobalScope")}} interface repeatedly calls a function or executes a code snippet, with a fixed time delay between each call.
+
+This method returns an interval ID which uniquely identifies the interval, so you can remove it later by calling {{domxref("WorkerGlobalScope.clearInterval", "clearInterval()")}}.
```
Comment (skyclouds2001) [inline files/en-us/web/api/workerglobalscope/setinterval/index.md:13]:
```suggestion
This method returns an interval ID which uniquely identifies the interval, so you can remove it later by calling {{domxref("WorkerGlobalScope.clearInterval", "clearInterval()")}}.

This method is also available in window contexts: for a detailed description and usage notes of `setInterval()`, see the {{domxref("Window.setInterval()")}} page.
```

File: files/en-us/web/api/workerglobalscope/setinterval/index.md (hunk 1 above)
Comment (skyclouds2001) [inline files/en-us/web/api/workerglobalscope/setinterval/index.md:62]:
```suggestion
```

File: files/en-us/web/api/workerglobalscope/setinterval/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+This method is also available in window contexts: for a detailed description and usage notes of `setInterval()`, see the {{domxref("Window.setInterval()")}} page.
```
Comment (wbamberg) [inline files/en-us/web/api/workerglobalscope/setinterval/index.md:15]:
```suggestion
This method is also available in window contexts: for a detailed description of `setInterval()`, see the {{domxref("Window.setInterval()")}} page.
```

("a detailed description and usage notes of `setInterval()`" isn't really grammatical, and just description is fine here.)

**Review by wbamberg — APPROVED** [review]
👍 nice work @skyclouds2001 . Only one more to go?

**Comment by skyclouds2001** [comment]
> 👍 nice work @skyclouds2001 . Only one more to go?

Yes

**Pull request merged**